spec/models/lap_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Lap, type: :model do
+  describe 'associations' do
+    it { should belong_to(:stopwatch) }
+  end
+
+  describe 'validations' do
+    it "is valid with a time, difference and stopwatch_id" do
+      user = User.create( email: 'test@gmail.com', name: 'Test', password: '123456')
```

Comment:
Try to avoid hardcoded values

Reply:
Sure, I forgot to implement factory in this model.